samples/drivers/CAN/prj.conf.mcp2515

```diff
@@ -0,0 +1,6 @@
+# CAN + MCP2515 config
```

Comment:
We should make prj.conf generic and use samples/drivers/CAN/boards directory for any board specific config. Probably not worth fixing in this PR (unless you really want to) but at least to be done later.